api/consumers.py

```diff
@@ -137,6 +139,13 @@ def fetch_current_question(self):
             return session.current_question.to_json()
         return None
 
+    def set_question_open(self, question: QuestionMultipleChoice):
+        quiz_session_question, created = QuizSessionQuestion.objects.get_or_create(
+            question=question, quiz_session__code=self.code
+        )
+        quiz_session_question.opened_at = timezone.now()
+        quiz_session_question.save()
```

Comment:
How do you feel about setting a `timezone.now` default value for `opened_at`? That way, I believe, you would be able to omit all this code.

When `session.served_questions.add(next_question)` (on Line 159) executes, a new row/object will be created and added to `QuizSessionQuestion` with the default `opened_at` value.

Reply:
Implemented this logic and removed the set_question_open method.